prisma/schema.prisma

```diff
@@ -6,13 +6,42 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider          = "postgresql"
+  url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  email    String @unique
+  id       Int              @id @default(autoincrement())
+  email    String           @unique
   password String
-}
\ No newline at end of file
+  profile  Profile?
+  hobbies  HobbiesOnUsers[]
+}
+
+model Profile {
+  id        Int    @id @default(autoincrement())
+  username  String
```

Comment:
You don't have to change this, but consider the scenario where you might want to allow users to login using their username instead of their email - having username on the profile model will make that more difficult. Not impossible, but it would certainly be easier if username was on the User model.

Just something to be aware of in future :)

Reply:
Noted. Thanks